lib/mongo/error/bulk_write_error.rb

```diff
@@ -59,11 +59,20 @@ def build_message
         errors = @result['writeErrors']
         return nil unless errors
 
-        fragment = errors.first(10).map do |error|
-          "[#{error['code']}]: #{error['errmsg']}"
-        end.join('; ')
+        fragment = ""
+        cut_short = false
+        errors.first(10).each_with_index do |error, i|
+          fragment += "; " if fragment.length > 0
+          fragment += "[#{error['code']}]: #{error['errmsg']}"
+          fragment += " -- #{error['errInfo'].to_json}" if error['errInfo']
```

Comment:
Can you please add a comment somewhere in this file that shows an example message that is generated in this way, i.e. one containing multiple errors and the double dashes.